### PR TITLE
Add completed to-do list

### PR DESCRIPTION
**Pull request by MananSoni42**

### CODE REVIEW

**Comment by Pro-Panda** [comment]
Is it ready for review ?

**Comment by MananSoni42** [comment]
Yes, it works satisfactorily

File: To-do-list/manan/main.py [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            path = input('Enter path of Audio File: ')
+            while not to_do_list.reminder_audio(path):
+                path = input('Enter path of Audio File: ')
+
+        if val == 11:
+            to_do_list.save()
+            break
+        print('')
+
+### --- MAIN --- ###
```
Comment (Pro-Panda) [inline To-do-list/manan/main.py]: Redundant comment.

**Comment by Pro-Panda** [comment]
Tested. Doesn't work for me.

Also, some comments:
- [ ] Update readme
- [ ] Folders named `manan` and `shivangi` don't make sense. Use one of them
- [ ] Lots of merge commits. I'll take care while squashing, but avoid them in future. Rebase instead
- [ ] Use `flake8` and `autopep8` to make your code readable
- [ ] Remove `.pyc` files. Add a `.gitignore` to fix them
- [ ] Use `config parser` with relevant and self expanatory options instead of taking user input in decimals as options. Don't add everything in one file.  Create separate modules for each of your option, and create functions in them.
- [ ]  Don't overdo adding modules and functions

**Comment by Pro-Panda** [comment]
Closing in favor of #9

**Pull request closed without merging**